main.py

```diff
@@ -1,10 +1,45 @@
+from email.policy import default  # noqa: F401
```

Comment:
why?

Reply:
Because an error Flake8
F401 'email.policy.default' imported but unused

Reply:
so why importing it?)

Reply:
It seems to me that this is PayCharm automatically added emails processing, I did not notice. you are right)